molecule/default/molecule.yml

```diff
@@ -0,0 +1,75 @@
+---
+dependency:
+  name: galaxy
+  options:
+    role-file: requirements.yml
+driver:
+  name: docker
+lint:
+  name: yamllint
+  # https://yamllint.readthedocs.io/en/latest/configuration.html#ignoring-paths
+  options:
+    config-data: "{extends: relaxed, rules: {line-length: {max: 200}}}"
+platforms:
+  - name: debian-php7
+    image: debian:stretch
+    privileged: true
+    # Adding some rm of systemctl because ansible try to use it by default in the container.
+    command: sh -c 'rm -f /bin/systemctl /sbin/initctl ; while sleep 3600; do :; done'
+    groups:
+      - tag_project_php
+      - tag_role_php
+      - tag_project_php
+      - tag_cycloid_io_true
+      - tag_env_test
+
+  - name: debian-php5
+    image: debian:jessie
+    privileged: true
+    # Adding some rm of systemctl because ansible try to use it by default in the container.
+    command: sh -c 'rm -f /bin/systemctl /sbin/initctl ; while sleep 3600; do :; done'
+    groups:
+      - tag_project_php
+      - tag_role_php
+      - tag_project_php
+      - tag_cycloid_io_true
+      - tag_env_test
```

Comment:
Could we add Ubuntu (LTS whatever) with PHP 7.2?
Fundshop is using it, and I had some joke about this role, so if we can ensure it works via our tests

Reply:
Same comment for RedHat, as we have conditional for it. If we aim to manage it, then let's add some tests for it; if not then let's remove the unnecessary code.

Reply:
I'm adding an Ubuntu image + tests.
I won't test Redhat as I don't want to invest time for OS we don't currently manage. It's still a good thing to keep the code to handle it, but we'll write tests when it will be really useful, or let someone else do a contrib for that :)

Reply:
Hum okay